addons/website/data/website_data.xml

```diff
@@ -648,37 +648,37 @@
             <field name="public" eval="True"/>
             <field name="name">s_product_list_default_image_1.jpg</field>
             <field name="type">url</field>
-            <field name="url">/website/static/src/img/snippets_demo/s_product_list_1.jpg</field>
+            <field name="url">/website/static/src/img/snippets_demo/furniture.jpg</field>
         </record>
         <record id="website.s_product_list_default_image_2" model="ir.attachment">
             <field name="public" eval="True"/>
             <field name="name">s_product_list_default_image_2.jpg</field>
             <field name="type">url</field>
-            <field name="url">/website/static/src/img/snippets_demo/s_product_list_1.jpg</field>
+            <field name="url">/website/static/src/img/snippets_demo/clothes.jpg</field>
         </record>
         <record id="website.s_product_list_default_image_3" model="ir.attachment">
             <field name="public" eval="True"/>
             <field name="name">s_product_list_default_image_3.jpg</field>
             <field name="type">url</field>
-            <field name="url">/website/static/src/img/snippets_demo/s_product_list_1.jpg</field>
+            <field name="url">/website/static/src/img/snippets_demo/books.jpg</field>
         </record>
         <record id="website.s_product_list_default_image_4" model="ir.attachment">
             <field name="public" eval="True"/>
             <field name="name">s_product_list_default_image_4.jpg</field>
             <field name="type">url</field>
-            <field name="url">/website/static/src/img/snippets_demo/s_product_list_1.jpg</field>
+            <field name="url">/website/static/src/img/snippets_demo/essentials_oils.jpg</field>
         </record>
         <record id="website.s_product_list_default_image_5" model="ir.attachment">
             <field name="public" eval="True"/>
             <field name="name">s_product_list_default_image_5.jpg</field>
             <field name="type">url</field>
-            <field name="url">/website/static/src/img/snippets_demo/s_product_list_1.jpg</field>
+            <field name="url">/website/static/src/img/snippets_demo/services.jpg</field>
         </record>
         <record id="website.s_product_list_default_image_6" model="ir.attachment">
             <field name="public" eval="True"/>
             <field name="name">s_product_list_default_image_6.jpg</field>
             <field name="type">url</field>
-            <field name="url">/website/static/src/img/snippets_demo/s_product_list_1.jpg</field>
+            <field name="url">/website/static/src/img/snippets_demo/multimedia.jpg</field>
```

Comment:
Note: you left the old image in the repo but it does not seems used anywhere... or should we left it there for "compatibility" @JKE-be ? We don't really ensure "theme" / "default" images to be kept anywhere else anyway... so I would simply remove it :grimacing: